test/runtime/runtime-python.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { execCommand } from "../blackbox/utils/Utils";
+import path from "path";
+
+
+jest.setTimeout(50000);
+
+test("Python runtime testing", async () => {
+//   The 'python ' command here
+
+    const compileCommand = `cd ${path.resolve(
+        __dirname,
+        "./runtime-python/test/"
+    )} && python3 main.py`;
+    await execCommand(compileCommand);
```

Comment:
Seems like this test fails because of the execCommand fails or exits with status code 1 i.e. indicating a failure 🤔